src/test/java/com/squareup/javapoet/UtilTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2015 Square, Inc.
```

Comment:
Changing copyright dates can invalidate the copyright.

On Sat, Jan 16, 2016, 11:06 AM Christian Stein <notifications@github.com>
wrote:

> In src/test/java/com/squareup/javapoet/UtilTest.java
> <https://github.com/square/javapoet/pull/403#discussion_r49935781>:
>
> > @@ -0,0 +1,73 @@
> > +/*
> > + * Copyright (C) 2015 Square, Inc.
>
> Should be a mass update for all files, imho.
> … <#msg-f:1523550719527131786_>
> On Sat, Jan 16, 2016 at 8:04 PM, Jesse Wilson <notifications@github.com>
> wrote: In src/test/java/com/squareup/javapoet/UtilTest.java <#403
> (comment)
> <https://github.com/square/javapoet/pull/403#discussion_r49935762>>: > @@
> -0,0 +1,73 @@ > +/* > + * Copyright (C) 2015 Square, Inc. 2016 — Reply to
> this email directly or view it on GitHub <
> https://github.com/square/javapoet/pull/403/files#r49935762>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/square/javapoet/pull/403/files#r49935781>.
>